src/status_im/transport/message/v1/protocol.cljs

```diff
@@ -104,8 +112,13 @@
 (defrecord MessagesSeen [message-ids]
   message/StatusMessage
   (send [this chat-id cofx]
-    (send cofx
-          {:chat-id chat-id
-           :payload this}))
+    (if config/pfs-encryption-enabled?
+      (send-direct-message
+       chat-id
```

Comment:
first argument on first line, cofx first, send-direct-message should use fx/defn